### PR TITLE
zellij: new, 0.25.0

### PR DESCRIPTION
**Pull request by OriginCode**
Topic Description
-----------------

Add package `zellij`

Package(s) Affected
-------------------

`zellij` v0.25.0

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`

### CODE REVIEW

**Comment by OriginCode** [comment]
LGTM on amd64:
圖片

**Comment by OriginCode** [comment]
`loongson3` FTBFS, unsupported platform
圖片

**Comment by OriginCode** [comment]
`ppc64el` FTBFS, unsupported platform
圖片

**Comment by OriginCode** [comment]
LGTM on arm64:
image

File: extra-utils/zellij/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,8 @@
+PKGNAME=zellij
+PKGSEC=utils
+PKGDEP=""
```
Comment (MingcongBai) [inline extra-utils/zellij/autobuild/defines]: Really?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**